.github/workflows/_linux-build.yml

```diff
@@ -230,6 +236,12 @@ jobs:
             DOCKER_SHELL_CMD=
           fi
 
+          if [[ ${MAX_JOBS_OVERRIDE} == "" ]]; then
+            MAX_JOBS="$(nproc --ignore=2)"
```

Comment:
The PR description implies you only want to change the setting for sm89 jobs, but doesn't this change the default for all jobs to be nproc - 2 instead of nproc - 4?